frontend/src/components/PopupWithForm/PopupSignup.js

```diff
@@ -132,7 +129,9 @@ if(isSuccess){
                 Sign in
               </span>
             </p>
+            <button onClick={props.onClose} className="popup__close-btn"></button>
```

Comment:
`          <button onClick={props.onClose} className="popup__close-btn"></button>
`

when I close popups with the `cross buttons` there is a `submit` action. You need to add ` type="button"` to any buttons that are **not** submit buttons

Reply:
`            <button onClick={props.onClose} className="popup__close-btn"></button>` is duplicated 3 times. You have it inside `PopupWithForm` already